test/common/bond_test.js

```diff
@@ -86,7 +86,7 @@ test(
                 'drs://drs.dev.singlecell.gi.ucsc.edu' +
                 '/bee7a822-ea28-4374-8e18-8b9941392723?version=2019-05-15T205839.080730Z'
             ),
-            BondProviders.default,
+            BondProviders.DCF_FENCE,
```

Comment:
Again, tweak the test name to explicitly state `dcf-fence` instead of `default` Bond Provider.